server/command.go

```diff
@@ -70,6 +70,58 @@ func (p *Plugin) handleHelp(args []string, extra *model.CommandArgs) (string, er
 	return p.getHelpText(), nil
 }
 
+func (p *Plugin) handleStartConfirm(args []string, extra *model.CommandArgs) (string, error) {
+	if len(args) > 1 {
+		return tooManyParametersText, nil
+	}
+	userID := extra.UserId
+	user, appErr := p.API.GetUser(userID)
+	if appErr != nil {
+		return "Cannot get user.", errors.Wrap(appErr, "cannot get user")
+	}
+	channelID := extra.ChannelId
+	channel, err := p.API.GetChannel(channelID)
+	if err != nil {
+		return "", err
+	}
+	message := ""
+	if channel.IsGroupOrDirect() {
+		var members *model.ChannelMembers
+		members, err = p.API.GetChannelMembers(channelID, 0, 100)
+		if err != nil {
+			return "", err
+		}
+
+		if members != nil {
+			p.API.LogDebug(fmt.Sprintf("%d members in channel %s", len(*members), channelID))
+			membersCount := len(*members)
+			message += "\n" + fmt.Sprintf("You are about a create a meeting in a channel with %d members", membersCount)
+		}
+
+	}
+
+	post := &model.Post{
+		UserId:    p.botUserID,
+		ChannelId: channelID,
+		Message:   message,
+		Type:      "custom_mstmeetings",
+		Props: map[string]interface{}{
+			"type":                     "custom_mstmeetings",
+			"meeting_status":           postTypeDialogWarn,
+			"meeting_personal":         true,
+			"meeting_creator_username": user.Username,
+			"meeting_provider":         msteamsProviderName,
+			"message":                  message,
+		},
+	}
+	_, appErr = p.API.CreatePost(post)
```

Comment:
Shouldn't this be an ephemeral message?

Reply:
yeah it should be.

Reply:
@larkox  been facing some issues with the ephemeral post. Will update you here